From 10419048721f7868c815b316e4d2e0a439422044 Mon Sep 17 00:00:00 2001
From: Dan McDonald <danmcd@joyent.com>
Date: Thu, 10 Jan 2019 15:30:39 -0500
Subject: [PATCH] OS-7504 varpd svp_remote_resolved exits on unowned mutex
 Reviewed by: Jason King <jbk@joyent.com> Reviewed by: Mike Zeller
 <mike.zeller@joyent.com> Reviewed by: Robert Mustacchi <rm@joyent.com>

---
 usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c b/usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c
index 5d19f8a388..4d10d1dba4 100644
--- a/usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c
+++ b/usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c
@@ -919,7 +919,6 @@ svp_conn_fallout(svp_conn_t *scp)
 		    "unkonwn state");
 	}
 	mutex_exit(&scp->sc_lock);
-	mutex_exit(&srp->sr_lock);
 }
 
 int
-- 
2.21.0

